src/main/java/net/ltxprogrammer/changed/latexvariant/CheckCondition.java

```diff
@@ -0,0 +1,64 @@
+package net.ltxprogrammer.changed.latexvariant;
```

Comment:
This should go under the `extensions` package alongside the other written compatibilities.